src/plugins/dashboard/public/dashboard_app/top_nav/use_dashboard_menu_items.tsx

```diff
@@ -200,12 +201,12 @@ export const useDashboardMenuItems = ({
         run: showShare,
       } as TopNavMenuData,
 
-      options: {
-        ...topNavStrings.options,
-        id: 'options',
-        testId: 'dashboardOptionsButton',
+      settings: {
+        ...topNavStrings.settings,
+        id: 'settings',
+        testId: 'dashboardSettingsButton',
         disableButton: isSaveInProgress,
```

Comment:
Good idea. I think all the nav bar items should be disabled while the flyout is open since the flyout contains local state that could be lost by clicking "Share", "Save as", or  "Switch to view mode".